spec/controllers/api/v1/workflows_controller_spec.rb

```diff
@@ -734,6 +734,28 @@
     end
   end
 
+  describe '#unretire_subjects' do
+    before do
+      default_request scopes: scopes, user_id: authorized_user.id
+    end
+
+    let(:subject_set_project) { project }
+    let(:subject_set) { create(:subject_set, project: project, workflows: [project.workflows.first]) }
```

Comment:
not blocking - fyi you can setup a subject_set that will create some subjects automatically for you, https://github.com/zooniverse/panoptes/blob/61d184848bfe367f4536ddbe3054988c3e734741/spec/factories/subject_sets.rb#L19
e.g. 
```
let(:subject_set) do 
  create(:subject_set_with_subjects, num_subjects: 1, project: project, workflows: [project.workflows.first])
end